City-of-Helsinki/drupal-employment-services
# THF-585: THF-585: Add more roles to editor

Added role editor permission to set published on date.

### Testing

- Log in as Editor Role
- Create content --> one for each content type.
- Leave the content unpublished and put the `published on date `to later the same date.
- Publish the content
- Take private window and check if you can see the page without logging in.
